gate-core/src/main/groovy/com/netflix/spinnaker/gate/config/ServiceConfiguration.groovy

```diff
@@ -18,20 +18,32 @@ package com.netflix.spinnaker.gate.config
 
 import groovy.transform.CompileStatic
 import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
 import org.springframework.boot.context.properties.ConfigurationProperties
 import org.springframework.context.ApplicationContext
 import org.springframework.stereotype.Component
 
+import javax.annotation.PostConstruct
+
 @CompileStatic
 @Component
 @ConfigurationProperties
 class ServiceConfiguration {
+  List<String> healthCheckableServices
   List<String> discoveryHosts
   Map<String, Service> services = [:]
 
   @Autowired
   ApplicationContext ctx
 
+  @PostConstruct
+  void postConstruct() {
+    // this check is done in a @PostConstruct to avoid Spring's list merging in @ConfigurationProperties (vs. overriding)
+    healthCheckableServices = healthCheckableServices ?: [
```

Comment:
The yaml override did not work when I had this as `List<String> healthCheckableServices = ["orca", "clouddriver"]`.

Rather it would merge what was in the yaml with `["orca", "clouddriver"]`. Seemed backwards but I don't think this is the first time we've seen this.